### PR TITLE
Implement flexBasis

### PR DESCRIPTION
**Pull request by rborn**
This should address https://github.com/jaredh159/tailwind-react-native-classnames/issues/202  😊

### CODE REVIEW

**Comment by jaredh159** [comment]
about the lint failure, if you run `npm run lint:fix` and `npm run format`, those will auto-fix.

also, if you want to do a "dry-run" of what approximates the CI tests and checks, run `npm run npub:precheck`

**Comment by rborn** [comment]
@jaredh159 ok, done, the old  tailwind issue is still there though  :)

**Comment by jaredh159** [comment]
@rborn tests all passed... what exactly are you referring to about the tailwind plugin?  were you getting a test failure locally? or trying to use the new feature?  can you elaborate?

File: supported-utilities.md [hunk 1]
```diff
@@ -45,6 +45,7 @@
 - 🚨 flex (RN implementation does not match web/tailwindcss)
 - 🚨 flexBasis
 - ✅ flexDirection
+- ✅ flexBasis
```
Comment (jaredh159) [inline supported-utilities.md:48]: can you also delete line 46 above?

**Comment by rborn** [comment]
Sorry I missed that line 😅

About the test error, I get this  when running tests locally 

Screenshot 2022-12-10 at 17 57 08

**Comment by jaredh159** [comment]
i'm pretty confused about your test/typescript error. it's not happening for me locally, or on CI, and you didn't touch that code at all. the tailwind version of that function is untyped javascript, so i'm not sure why there would be an issue. i wonder if there is something unusual about your local setup.

anyway, i'm comfortable merging this. thanks for the contribution. i'll cut a release in a few minutes. 👍

**Comment by jaredh159** [comment]
`v3.5.0` now released. thanks again.

**Pull request merged**